src/activate/activate.c

```diff
@@ -332,33 +338,33 @@ static int help(void) {
                "     --fdname=NAME[:NAME...] Specify names for file descriptors\n"
                "     --inetd                 Enable inetd file descriptor passing protocol\n"
                "\nNote: file descriptors from sd_listen_fds() will be passed through.\n"
-               "\nSee the %s for details.\n"
-               , program_invocation_short_name
-               , link
-        );
+               "\nSee the %s for details.\n",
+               program_invocation_short_name,
+               link);
 
         return 0;
 }
 
 static int parse_argv(int argc, char *argv[]) {
-        enum {
+        enum
+        {
                 ARG_VERSION = 0x100,
                 ARG_FDNAME,
                 ARG_SEQPACKET,
                 ARG_INETD,
         };
 
         static const struct option options[] = {
-                { "help",        no_argument,       NULL, 'h'           },
-                { "version",     no_argument,       NULL, ARG_VERSION   },
-                { "datagram",    no_argument,       NULL, 'd'           },
-                { "seqpacket",   no_argument,       NULL, ARG_SEQPACKET },
-                { "listen",      required_argument, NULL, 'l'           },
-                { "accept",      no_argument,       NULL, 'a'           },
-                { "setenv",      required_argument, NULL, 'E'           },
-                { "environment", required_argument, NULL, 'E'           }, /* legacy alias */
-                { "fdname",      required_argument, NULL, ARG_FDNAME    },
-                { "inetd",       no_argument,       NULL, ARG_INETD     },
+                { "help", no_argument, NULL, 'h' },
+                { "version", no_argument, NULL, ARG_VERSION },
+                { "datagram", no_argument, NULL, 'd' },
+                { "seqpacket", no_argument, NULL, ARG_SEQPACKET },
+                { "listen", required_argument, NULL, 'l' },
+                { "accept", no_argument, NULL, 'a' },
+                { "setenv", required_argument, NULL, 'E' },
+                { "environment", required_argument, NULL, 'E' }, /* legacy alias */
+                { "fdname", required_argument, NULL, ARG_FDNAME },
+                { "inetd", no_argument, NULL, ARG_INETD },
```

Comment:
hmm, I much prefer aligning things as tables when in fact they really *are* tables conceptually, like in this case. Isn't there a way to make llvm understand this for "static const struct" arrays like this one?

Reply:
there are two ways around this:
1. mark block with clang-format ignore (not so desirable)
2. use comments to apply aligning
for example the block above could be written the following way, which stays stable with clang-format:
```cpp
static const struct option options[] = {
                { "help" /*********/, no_argument /***/, NULL, 'h' },
                { "version" /******/, no_argument /***/, NULL, ARG_VERSION },
                { "datagram" /*****/, no_argument /***/, NULL, 'd' },
                { "seqpacket" /****/, no_argument /***/, NULL, ARG_SEQPACKET },
                { "listen" /*******/, required_argument, NULL, 'l' },
                { "accept" /*******/, no_argument /***/, NULL, 'a' },
                { "setenv" /*******/, required_argument, NULL, 'E' },
                { "environment" /**/, required_argument, NULL, 'E' }, /* legacy alias */
                { "fdname" /*******/, required_argument, NULL, ARG_FDNAME },
                { "inetd" /********/, no_argument /***/, NULL, ARG_INETD },
                {}
        };
```
This is quite readable in most editors.